src/styles/global/print.scss

```diff
@@ -1,8 +1,96 @@
 //
 // CUSTOMIZED PRINT STYLES
```

Comment:
Please add a general description like "apply general print styles ... for PDF creation ... apply to pages with a print icon"